xml/obs_best_practice_integrate_scm_sources.xml

```diff
@@ -0,0 +1,128 @@
+        <screen>
+          <command>
+            &lt;services&gt;
+		  &lt;service name="obs_scm"&gt;
+		    &lt;param name="url"&gt;https://github.com/FreeCAD/FreeCAD.git&lt;/param&gt;
+		    &lt;param name="scm"&gt;git&lt;/param&gt;
+		  &lt;/service&gt;
+
+		  &lt;service name="set_version" mode="buildtime"/&gt;
+		  &lt;service name="tar" mode="buildtime"/&gt;
+		  &lt;service name="recompress" mode="buildtime"&gt;
+		    &lt;param name="file"&gt;*.tar&lt;/param&gt;
+		    &lt;param name="compression"&gt;xz&lt;/param&gt;
+		  &lt;/service&gt;
+		&lt;/services&gt;
+          </command>
+        </screen>
+        This will create an obscpio archive via the obs_scm service with the latest sources. 
+        This archive will get extracted at build time and be processed via the other services
+        to build a compressed tar ball for rpmbuild. 
+        To follow a specific branch and additional parameter for "revision" is needed for the
+        obs_scm service.
+       </para>
+     </sect2>
+     <sect2>
+       <title>Fixed versions</title>
+       <para>
+         You may want to build an archive for a fixed version, for example an official release
+         which has been tagged by the upstream project. It is recommend to specify the 
+         mode="disabled" and to submit the archive via the following
+        <screen>
+          <command>
+	    # osc service runall
+	    # osc ar
+	    # osc commit
+          </command>
+        </screen>
+        commands.
+       </para>
+     </sect2>
+     <sect2>
+       <title>Avoid tar balls</title>
+       <para>
+         tar balls are not a requirement by &obsa;, but by the packaging tool, eg. rpmbuild. However,
+         you may want to decide not to ship a tar ball inside of the src.rpm. This makes sense for
+         large sources where the compression time and needed disk space is just considered a waste
+         for short living builds. You can simplify your _service file in that case, but you need to help rpmbuild to work directly in the source.
+         It is also a good practice to package the _service file instead of the tar ball to 
+         give the user a chance to rebuild the src.rpm as long the external SCM server is
+         providing the sources. The simplified _service file looks like this:
+        <screen>
+          <command>
+            &lt;services&gt;
+		  &lt;service name="obs_scm"&gt;
+		    &lt;param name="url"&gt;https://github.com/FreeCAD/FreeCAD.git&lt;/param&gt;
+		    &lt;param name="scm"&gt;git&lt;/param&gt;
+		  &lt;/service&gt;
+
+		  &lt;service name="set_version" mode="buildtime"/&gt;
+            &lt;/services&gt;
+          </command>
+        </screen>
+        The spec file needs some hints to build inside the extracted sources directly. The macro
+        can be used to switch to build tar balls or not to keep it working for stable releases where
+        you want to provide a complete source rpm.
+        <screen>
+          <command>
```

Comment:
* Remove `command`.
* Remove any indendation, start content of `screen` always at first column.